### ISSUE TITLE
Installation Documentation improvement

### ISSUE THREAD

**Issue by vaibhavsingh97** [OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io#119]
As of now new users can easily understand how to install this webapp locally but if they want to deploy the application in heroku they can't because of lack of explanation. We could have something like this: 
![image](https://user-images.githubusercontent.com/8705386/30612369-d357d154-9d73-11e7-8b82-044f304ab97d.png)
More organized documentation and proper steps to deploy locally and on heroku.
In Documentation details of `technology stack`, `branch policies`, `Contributions best practices` and `License details` are missing.

**Comment by vaibhavsingh97** [comment]
@tapasweni-pathak I can raise seperate issues for these and these issues can be fixed by first time contributor.

**Comment by vaibhavsingh97** [comment]
@anubhakushwaha @nikhita Please share your opinion 😄

**Comment by nikhita** [comment]
@vaibhavsingh97 I :heart: it!

**Comment by vaibhavsingh97** [comment]
@nikhita What i am thinking to have a gist/Google doc where everyone can collaborate and can improve installation documentation to more beginer friendly.

**Comment by nikhita** [comment]
@vaibhavsingh97 Sounds great. I think we should go with a google doc since it has better sharing features.

Just to be clear, the google doc will be kind of a "draft", right? We'll later port it to markdown?

**Comment by vaibhavsingh97** [comment]
@nikhita yeah, It's just a draft, We would port that draft in markdown. I will share the link of doc in a minute

**Comment by devesh-verma** [comment]
@vaibhavsingh97 can I take this one?
